docs/semgrep-cloud-platform/gitlab-mr-comments.md

```diff
@@ -22,58 +26,81 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Enabling GitLab merge request comments
+# Set up GitLab merge request comments
+
+Semgrep can create **merge request (MR) comments** in your GitLab repository. These comments inform your developers of **findings**, such as security issues, in their PRs. Semgrep can also provide remediation tips or code fixes that your developers can click to **commit** into their code directly.
 
 :::info Prerequisites
-* Pull request (PR) comments can only be enabled through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up Slack notifications.
-* To receive alerts and notifications, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+- Merge request (MR) comments can only be set up through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
+- You must connect your GitLab organization (org) to Semgrep. 
+    - For GitLab Cloud users, this is done automatically after signing in.
+    - For GitLab self-managed users, go to [<i class="fas fa-external-link fa-xs"></i> Settings](https://semgrep.dev/orgs/-/settings) and click **Add GitLab Self-Managed**.
+- You must add or onboard a Semgrep project (repository) to SCP and it must complete at least one full scan on your default branch successfully.
 :::
 
-This section documents how to enable Semgrep Cloud Platform to post comments on merge requests.
+## Conditions for PR comment creation
```

Comment:
I just realized that all instances of PR in this doc should probably be changed to MR; maybe you could leverage `{props.request_type}` or something in the components?

Reply:
Yes, this is a great catch. I try not to use the props too much as it's a [little limited in what it can do](https://www.notion.so/semgrep/TW-Handbook-90a5316973be47c193a0a2ed33d3cabe?pvs=4#60a208ce84ca489fa0c8ba0b8c1cf306), but I hope my edits are still understandable. Thank you Katie.